samples/subsys/llext/modules/sample.yaml

```diff
@@ -0,0 +1,34 @@
+common:
+  tags: llext
+  arch_allow:
+    - arm
+    - xtensa
+  filter: not CONFIG_MPU and not CONFIG_MMU and not CONFIG_SOC_SERIES_S32ZE
+  platform_exclude:
+    - apollo4p_evb
+    - apollo4p_blue_kxr_evb
```

Comment:
Please add a comment referring to issues 72775 / 73443 for apollo platforms, to make the reason why they are excluded obvious (as done for the numaker right below).